playground/mocks/data/api/v1/authn/forced-idp-discovery-linkedin-idp.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Just curious why those 2 responses structure are different (e.g. no `stateToken`, `_links` is array instead of object.